tests/bootstrap/test_cli.py

```diff
@@ -180,8 +180,11 @@ def test_container_starts_non_root(self, runner, monkeypatch, container_client):
         output = container_client.exec_in_container(config.MAIN_CONTAINER_NAME, ["ps", "-u", user])
         assert "supervisord" in to_str(output[0])
 
+    @pytest.mark.skip(reason="skip temporarily until compatible localstack image is on DockerHub")
```

Comment:
re-enable as soon as the new Docker image is on DockerHub.
Bumping the `docker` Python dependency requires a synchronized update of LS, ext, and the Docker image on DockerHub. The current CLI tests use the latest image from DockerHub.

The update of `docker` adds a new `platform` flag and running this test against an old localstack image yields the error `TypeError: run() got an unexpected keyword argument 'platform'` (not visible in CI). Locally building a new LS Docker image via `make docker-build` fixes the issue.